src/Diagnostics/atmos_default.jl

```diff
@@ -34,268 +36,261 @@ function atmos_default_init(dgngrp::DiagnosticsGroup, currtime)
-    ds.qt_sgs += MH * ha.qt_sgs
-    ds.ht_sgs += MH * ha.ht_sgs
-
-    # vertical fluxes
-    ds.vert_eddy_mass_flux += MH * (w - w̃) * (state.ρ - ha.ρ)
-    ds.vert_eddy_u_flux += MH * (w - w̃) * (u - ha.ρu / ha.ρ)
-    ds.vert_eddy_v_flux += MH * (w - w̃) * (v - ha.ρv / ha.ρ)
-    ds.vert_eddy_ql_flux += MH * (w - w̃) * (th.q_liq - ha.q_liq)
-    ds.vert_eddy_qv_flux += MH * (w - w̃) * (th.q_vap - ha.q_vap)
-    ds.vert_eddy_thd_flux += MH * (w - w̃) * (th.θ_dry - ha.θ_dry)
-    ds.vert_eddy_thv_flux += MH * (w - w̃) * (th.θ_v - ha.θ_v)
-    ds.vert_eddy_thl_flux += MH * (w - w̃) * (th.θ_liq_ice - ha.θ_liq_ice)
-    # TODO: temporary fix
-    if isa(atmos.moisture, EquilMoist)
-        ds.vert_eddy_qt_flux += MH * (w - w̃) * (q_tot - q̃_tot)
-        ds.vert_qt_flux += MH * w * q_tot
+# Variances and covariances
+function vars_atmos_default_ho(m::AtmosModel, FT)
+    @vars begin
+        var_u::FT               # u′u′
+        var_v::FT               # v′v′
+        var_w::FT               # w′w′
+        w3::FT                  # w′w′w′
+        tke::FT
+        var_ei::FT              # e_int′e_int′
+
+        cov_w_u::FT             # w′u′
+        cov_w_v::FT             # w′v′
+        cov_w_rho::FT           # w′ρ′
+        cov_w_thd::FT           # w′θ_dry′
+        cov_w_thv::FT           # w′θ_v′
+        cov_w_ei::FT            # w′e_int′
+
+        moisture::vars_atmos_default_ho(m.moisture, FT)
     end
+end
+vars_atmos_default_ho(::MoistureModel, FT) = @vars()
+function vars_atmos_default_ho(m::EquilMoist, FT)
+    @vars begin
+        var_qt::FT              # q_tot′q_tot′
+        var_thl::FT             # θ_liq_ice′θ_liq_ice′
+
+        cov_w_qt::FT            # w′q_tot′
+        cov_w_ql::FT            # w′q_liq′
+        cov_w_qv::FT            # w′q_vap′
+        cov_w_thl::FT           # w′θ_liq_ice′
+        cov_qt_thl::FT          # q_tot′θ_liq_ice′
+        cov_qt_ei::FT           # q_tot′e_int′
+    end
+end
+num_atmos_default_ho_vars(m, FT) = varsize(vars_atmos_default_ho(m, FT))
+atmos_default_ho_vars(m, array) =
+    Vars{vars_atmos_default_ho(m, eltype(array))}(array)
 
-    # variances
-    ds.uvariance += MH * (u - ũ)^2
-    ds.vvariance += MH * (v - ṽ)^2
-    ds.wvariance += MH * (w - w̃)^2
-
-    # skewness
-    ds.wskew += MH * (w - w̃)^3
+function compute_ho_sums!(
+    atmos::AtmosModel,
+    state,
+    diffusive_flux,
+    aux,
+    thermo,
+    MH,
+    ha,
+    sums,
+)
+    u = state.ρu[1] / state.ρ
+    u′ = u - ha.u
+    v = state.ρu[2] / state.ρ
+    v′ = v - ha.v
+    w = state.ρu[3] / state.ρ
+    w′ = w - ha.w
+    e_int′ = thermo.e_int - ha.ei
+    θ_dry′ = thermo.θ_dry - ha.thd
+    θ_vir′ = thermo.θ_vir - ha.thv
+
+    sums.var_u += MH * state.ρu[1] / state.ρ * state.ρu[1]
+    sums.var_v += MH * state.ρu[2] / state.ρ * state.ρu[2]
+    sums.var_w += MH * state.ρu[3] / state.ρ * state.ρu[3]
+    sums.w3 += MH * state.ρu[3] / state.ρ * state.ρu[3] / state.ρ * state.ρu[3]
```

Comment:
Why are these here? (221-224 are the correct definition of var_u, var_v, var_w, and w3.)